docs/topics/mta-rn-new-features-7-0-0.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-7-0-0_{context}"]
+= New features
+
+
+This section describes the new features and improvements of the {ProductName} ({ProductShortName}) 7.0.0.
+
+.Red Hat Single Sign On (RHSSO) configuration
+
+{ProductShortName} supports Red Hat Single Sign-On (RHSSO) configuration, to enable adding users' roles by using `keycloak` to control {ProductShortName} authorization mechanisms. See link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.0/html/user_interface_guide/mta-6-installing-web-console-on-openshift_user-interface-guide#red_hat_single_sign_on[Red Hat Single Sign-On]. 
+
+
+.YAML syntax for new rules
+
+New rules that use YAML syntax to support metadata, `message` and `tag` actions, rule conditions, provider conditions for `java` and `go` providers, and other file tagging and characteristics.
+
+
+.Filename of downloaded report file matches application name
+
+Downloaded files are now named according to the migrated application.
+
+
+.Tag category menu shows only categories relevant to the source
+
+The *Tag category* drop-down list now shows only tag categories that are relevant to the source.
+
+
+.Multi-language analysis
+
+{ProductShortName} 7.0.0 supports migrating applications written in languages implemented in Language Server Protocol (LSP).
```

Comment:
There are two languages at this time . Java and Go out of which Go is Tech Preview only.

Reply:
Added note regarding Go in tech preview.